docusaurus/docs/Android/04-compose/04-message-components/02-message-list-header.mdx

```diff
@@ -120,19 +121,20 @@ fun MessageListHeader(
             onBackPressed = onBackPressed,
         )
     },
```

Comment:
Above should be `NetworkLoadingIndicator` not View

Reply:
We should also pull out the BackButton as a `DefaultMessageListHeaderLeadingContent`, right?

Same goes for other slots.

Reply:
Finally - we are missing the new parameters you added here in docs - color and shape.